main.go

```diff
@@ -0,0 +1,7 @@
+package main
+
+import mpawsbilling "github.com/syou6162/check-docker-ps/lib"
```

Comment:
Typo? 🤔 　`github.com/tjinjin/mackerel-plugin-aws-billing/lib`

Reply:
exactly 🙇  I fixed it!